stdlib: support get current working directory (#5276)
bpftrace/bpftrace: docs/stdlib.md

## Helpers

### abspath

> - `void abspath(string path, uint64 max_path_depth);`

Reviewer: nit
```suggestion
- `void abs_path(string path, uint64 max_path_depth);`
```

Takes a string-like variable, and converts to an absolute path.

In fact, this function concatenates the program's current working directory and the filename to form the absolute path of the file. If an empty string is passed in, the current working directory can be obtained.

For example:
```
let $cwd : string[1024];
abspath($cwd, 32);

> printf("%s\n", $cwd);

Reviewer: Can you show an example of what this prints?

## Helpers

### abs_path
- `string abs_path(string filename, uint64 max_path_depth);`

Takes a string-like variable, and converts to an absolute path.

In fact, this function concatenates the program's current working directory and the filename to form the absolute path of the file. If an empty string is passed in, the current working directory can be obtained.

> For example:

Reviewer: ```suggestion
- `string abs_path(string path, uint64 max_path_depth);`

Returns the absolute path for the passed relative path or filename . If an empty string is passed in, the current working directory is returned.

For example:
```

## Helpers

### abs_path
- `string abs_path(string path, uint64 max_path_depth);`

Takes a string-like variable, and converts to an absolute path.

> Returns the absolute path for the passed relative path or filename . If an empty string is passed in, the current working directory is returned.

Reviewer: ```suggestion
Returns the absolute path for the passed relative path or filename. If the `path` param is an empty string then the current working directory is returned.
```

> For example:

Reviewer: ```suggestion
Example:
```

```
$ cat open.bt
tracepoint:syscalls:sys_enter_openat {
  print(str(args.filename));
}
```
The test procedure is `(1)->(2)->(3)->(4)`:
```
(1) $ cd /home/rongtao/
(3) $ vim .bashrc
(2) $ sudo ./open.bt
(4) .bashrc
```
Use `abs_path()` to get the absolute path of a file:
```
tracepoint:syscalls:sys_enter_openat {
  print(abs_path(str(args.filename), 32));
}
```
Then:
```
(4) /home/rongtao/.bashrc

> ```

Reviewer: ```suggestion
tracepoint:syscalls:sys_enter_openat {
  print(abs_path(str(args.filename), 32));
}
```
And then running a  command like `vim .bashrc` will print:
```
/home/rongtao/.bashrc
```

This utilizes the BPF helper `get_current_task`

### cwd
- `string cwd();`
- `string cwd;`

get current working directory

> see also `abs_path()`.

Reviewer: ```suggestion
Get the current working directory. This is equivalent to calling `abs_path()` with an empty string.
```
